frontend/src/utils.js

```diff
@@ -99,12 +99,7 @@ function findMetric(href) {
 }
 
 function isFile(id) {
-  return ![
-    'referrer',
-    'totalDownloads',
-    'totalUploads',
-    'testpilot_ga__cid'
-  ].includes(id);
+  return /^[0-9a-fA-F]{10}$/.test(id);
```

Comment:
micro nit, but you could just pass the `i` flag to make it case insensitive so you wouldnt need the extra "A-F" character range. 🤷‍♀️ 

```js
return /^[0-9a-f]{10}$/i.test(id);
```

But, yeah, well, that's just, like, my opinion, man.